Modules/Editors/ProjectCenter/PCEditor.m

```diff
@@ -948,21 +939,30 @@ - (void)textDidChange:(NSNotification *)aNotification
     }
 }
 
-- (void)textViewDidChangeSelection:(NSNotification *)notification
+- (NSRange)textView:(NSTextView *)textView
+willChangeSelectionFromCharacterRange:(NSRange)oldSelectedCharRange
+   toCharacterRange:(NSRange)newSelectedCharRange
 {
-  id object;
-
-  object = [notification object];
+  NSLog(@"Will change selection from %@ to %@", NSStringFromRange(oldSelectedCharRange), NSStringFromRange(newSelectedCharRange));
 
+  NSLog(@"is pressing key %d", editorTextViewIsPressingKey);
   if (editorTextViewIsPressingKey == NO)
     {
-      id object;
-
-      object = [notification object];
-      if (object == _intEditorView || object == _extEditorView)
-	[self computeNewParenthesisNesting: object];
+      // unhighlight also invalidates old locations
+      if (textView == _intEditorView || textView == _extEditorView)
+	[self unhighlightCharacter: textView];
     }
 
+  return newSelectedCharRange;
+}
+
+- (void)textViewDidChangeSelection:(NSNotification *)notification
```

Comment:
How calls this method? You no longer listen to that notification.

Reply:
it is called because part of the NSTextViewDelegate - and ev is set to be its own delegate.